.npm/.scripts/prepare-wasm-packages.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+NPM_DIR=${ROOT_DIR}/.npm
+PKGS_DIR=${NPM_DIR}/packages
+
+
+write_template ()
+{
+  NAME_DASHED=$1
+  NAME_UNDERSCORED=$2
+  TEMPLATE=$3
+
+  PKG_TEMPLATE_DIR=${NPM_DIR}/.scripts/template
+  PKG_DIR=${PKGS_DIR}/${NAME_DASHED}
+
+  PKG_NAME=$(echo "${NAME_DASHED}" | sed -e 's/fuel-//g')
+  PKG_VERSION=$(cat ${ROOT_DIR}/Cargo.toml | sed -nr 's/^version = "([^"]+)"/\1/p')
+
+  echo "$(
+    cat ${PKG_TEMPLATE_DIR}/${TEMPLATE} \
+    | sed -e "s/{{NAME_DASHED}}/${NAME_DASHED}/g" \
+    | sed -e "s/{{NAME_UNDERSCORED}}/${NAME_UNDERSCORED}/g" \
+    | sed -e "s/{{PKG_NAME}}/${PKG_NAME}/g" \
+    | sed -e "s/{{PKG_VERSION}}/${PKG_VERSION}/g"
+  )" > ${PKG_DIR}/${TEMPLATE}
+
+}
+
+
+build_wasm_npm_pkg_for ()
+{
+  NAME_DASHED=$1
+  NAME_UNDERSCORED=$(echo "${NAME_DASHED}" | sed -e 's/-/_/g')
+
+  PKG_DIR=${PKGS_DIR}/${NAME_DASHED}
+
+  rm -rf ${PKG_DIR}/{src,dist}
+
+  cd ${ROOT_DIR}
+  cargo rustc -p ${NAME_DASHED} --target wasm32-unknown-unknown --features typescript --crate-type=cdylib --release
+  wasm-bindgen --target web ./target/wasm32-unknown-unknown/release/${NAME_UNDERSCORED}.wasm --out-dir ${PKG_DIR}/src
```

Comment:
Do we need to specify the typescript output type(maybe they will change the default behavior in the future)?

```suggestion
  wasm-bindgen --typescript --target web ./target/wasm32-unknown-unknown/release/${NAME_UNDERSCORED}.wasm --out-dir ${PKG_DIR}/src
```

Reply:
@xgreenx Nice catch!